sass/site/primary/_posts-and-pages.scss

```diff
@@ -149,6 +149,37 @@
 	}
 }
 
+// "Snipe" across top corner of cards for "Sold" vehicles
+.card.type-vehicle.tag-sold,
+.tag-sold .entry-content .ngg-gallery-thumbnail {
+	
+	a {
+  	display: block;
+  	position: relative;
+  	overflow: hidden;
+  	
+  	&::after {
+    	background-color: #b5161c;
+    	color: #fff;
+    	font-size: 1.125rem;
+    	font-weight: bolder;
+    	content: 'SOLD';
+    	letter-spacing: 1px;
+    	line-height: 1;
+    	padding: 0.375rem 6rem;
+    	position: absolute;
+    	top: 1.5rem;
+    	right: -5rem;
+    	transform: rotate(45deg);
+    }
+  }
+}
+
+.tag-sold .entry-content .ngg-gallery-thumbnail a::after {
```

Comment:
this is very high specificity. I'm wondering if it's really necessary, but it's also fine.

Reply:
I know, i'm not crazy about it but I didn't also want to go through the work of creating a specific template or PHP handler for the sold tag

Reply:
hmm yeah I think PHP would be more appropriate than a pseudo-element, but I see why you don't wanna bother.
I'd like `.tag-sold .ngg-gallery-thumbnail a::after` would work to simplify the selector, but 🤷‍♀